tests/PHPStan/Analyser/nsrt/conditional-types.php

```diff
@@ -68,7 +68,7 @@ public function testMicrotime(): void
 	}
 
 	/**
-	 * @return ($version is 8 ? true : ($version is 10 ? true : false))
+	 * @return ($version is 8|10 ? true : false)
```

Comment:
please don't change unrelated tests

Reply:
Sure, I can undo. I changed it because it's the example I copied from and then @mvorisek kindly suggested a worthy edit so going forward I didn't want others to make the same mistake I made.

Reply:
yeah thats true for stub declarations. but not for tests in which all kind of scenarios want to be tested